src/ontology/mondo-edit.obo

```diff
@@ -455486,6 +455489,15 @@ intersection_of: disease_has_feature NCIT:C68749 ! HER2/Neu Negative
 property_value: seeAlso https://github.com/monarch-initiative/mondo/issues/4933 xsd:string
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700080
+name: EPHB4-associated vascular malformation spectrum
+def: "Any vascular malformation in which the cause of the disease is a variation in the EPHB4 gene. Variants in the EPHB4 gene have been observed in individuals with capillary malformation-arteriovenous malformation (CM-AVM) (characterized by the presence of multiple small capillary malformations, mostly on the face and limbs, +/- other arteriovenous malformations or arteriovenous fistulas) as well as with lymphatic malformations, ranging in severity from severe non-immune hydrops fetalis to varicose veins and/or subclinical lymphatic anomalies, even within the same family (PMID: 27400125). Evidence suggests that both of these presentations are caused by loss of function, though the exact mechanism by which this occurs is variable; some variants demonstrate reduced expression and defects in subcellular localization with aggregates, others have normal expression levels but reduced tyrosine kinase activity) (PMID: 33864021). It has been hypothesized that the different presentations may be due to differences in forward vs. reverse signaling defects, but this remains to be elucidated." [MONDO:patterns/disease_series_by_gene, PMID:27400125, PMID:2868770, PMID:30760892, PMID:33864021]
```

Comment:
@sabrinatoro for some reason I can't edit this branch myself. We shouldn't have the PMID in the text definition itself, just in the dbxref. Would you mind revising that? Everything else looks good, thanks!

Reply:
@nicolevasilevsky I can remove them, though I am questioning whether this whole part of the "definition" release belongs to the definition... Maybe it should be a comment about this term?

Reply:
I agree- that seems appropriate for a comment. Thanks!

Reply:
Thank you! I updated the definition and comment